### PR TITLE
Working version with latest SKL

### PR DESCRIPTION
**Pull request by kotylamichal**
Update to work with the latest version of SKL. Including necessary GRUB changes

### CODE REVIEW

File: dynamic-layers/openembedded-layer/recipes-support/skl/skl_git.bb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+DESCRIPTION = "Landing zone"
```
Comment (TomaszAIR) [inline dynamic-layers/openembedded-layer/recipes-support/skl/skl_git.bb:1]: @kotylamichal Secure Kernel Loader

File: dynamic-layers/openembedded-layer/recipes-support/skl/skl_git.bb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+inherit deploy
+
+do_deploy() {
+    install -d ${DEPLOYDIR}
+    install -m 0600 ${S}/skl.bin ${DEPLOYDIR}
+}
+
+FILES:${PN} += "${bindir}/skl /boot"
```
Comment (TomaszAIR) [inline dynamic-layers/openembedded-layer/recipes-support/skl/skl_git.bb:40]: @kotylamichal is `/boot` needed here?

File: dynamic-layers/openembedded-layer/recipes-support/skl/skl_git.bb (hunk 1 above)
Comment (kotylamichal) [reply dynamic-layers/openembedded-layer/recipes-support/skl/skl_git.bb:1]: done

File: dynamic-layers/openembedded-layer/recipes-support/skl/skl_git.bb (hunk 2 above)
Comment (kotylamichal) [reply dynamic-layers/openembedded-layer/recipes-support/skl/skl_git.bb:40]: Yes, it it used and it was before in older versions

**Review by TomaszAIR — APPROVED** [review]

**Pull request merged**